### ISSUE TITLE
Support JSON mutations

### ISSUE THREAD

**Issue by ospaarmann** [ospaarmann/exdgraph#9]
The Dgraph docs state that:

> Mutations can also be specified using JSON objects. This can allow mutations to be expressed in a more natural way. It also eliminates the need for apps to have custom serialisation code, since most languages already have a JSON marshalling library.
> 
> When Dgraph receives a mutation as a JSON object, it first converts in into multiple RDFs that are then processed as normal. 

That means mutations like this are possible:

```
{
  "name": "Alice",
  "friend": {
    "name": "Betty"
  }
}
```

Using Poison to decode maps could be a great way to support mutations.

Read more: https://docs.dgraph.io/mutations/#json-mutation-format

**Comment by ospaarmann** [comment]
Partially fixed by 061079a1cc569ad7a5052b66605671cad54073a9

**Comment by ospaarmann** [comment]
Fixed with 72a2f7c84c1e1411d18541902ca58d29cd4e3a47
